web/components/mod-report-item.tsx

```diff
@@ -110,7 +132,7 @@ const ModReportItem: React.FC<ReportItemProps> = ({
           />
         </Row>
         <Row className="mt-2 items-center">
-          Mod note: {''}
+          Mod note:&nbsp;
```

Comment:
for some reason {''} wasn't doing anything. I tried wrapping it in a span and that also didn't work.

Reply:
You need a space in between the quotes: `{' '}`

Reply:
Tried for far too long (10-minutes) to get {' '} to work but couldn't so leaving it as is. Why is &nbsp bad to use anyways? Alternatively could wrap the text in a div and add padding but that just seems worse.

Reply:
Ah, yeah, it doesn't matter that much, mostly it's for learning so you know how to do it better in the future.

You're right, Row, which has display `flex` is typically for containers of other containers, not for text.

I think I'd wrap mod note in a div like this `<div>Mode note</div>` and add `gap-2` to the Row parent.